src/SFML/Window/DRM/InputImplUDev.cpp

```diff
@@ -412,7 +412,7 @@ namespace
                             event.key.shift = shiftDown();
                             event.key.system = systemDown();
 
-                            keyMap[kb] = inputEvent.value;
+                            keyMap[static_cast<size_t>(kb)] = inputEvent.value;
```

Comment:
```suggestion
                            keyMap[static_cast<std::size_t>(kb)] = inputEvent.value;
```
We use the `std::` prefix for integer types ubiquitously now.